### PR TITLE
Timerange: Fixes a bug where custom time ranges didn't respect UTC

### LINKED ISSUE #18170: 6.3.0-beta1 custom timeranges does not seem to respect UTC
custom timerange doesn't seem to respect UTC. dashboard has UTC set, but when i select an UTC timerange, it seems to select the timerange in local time. (because the UTC times shown are wrong)
huh

### PR DESCRIPTION
**Pull request by kaydelaney**
**What this PR does / why we need it**:
Fixes a bug where custom time ranges didn't respect UTC

**Which issue(s) this PR fixes**:
Closes #18170

### CODE REVIEW

**Comment by kaydelaney** [comment]
Ah, apparently this stops relative times from displaying in the time picker - it won't display as "Last 7 days" but rather the time range that refers to. Looking into it now

File: packages/grafana-ui/src/components/TimePicker/TimePicker.tsx [hunk 1]
```diff
@@ -123,13 +124,21 @@ export class TimePicker extends PureComponent<Props, State> {
     const { isCustomOpen } = this.state;
     const options = this.mapTimeOptionsToSelectableValues(selectTimeOptions);
     const currentOption = options.find(item => isTimeOptionEqualToTimeRange(item.value, value));
-    const rangeString = rangeUtil.describeTimeRange(value.raw);
+
+    const isUTC = timeZone === 'utc';
+    const adjustedTo = isUTC ? value.from.utc() : value.from.local();
```
Comment (davkal) [inline packages/grafana-ui/src/components/TimePicker/TimePicker.tsx]: Ooops. Should be `value.to`.

**Review by davkal — CHANGES_REQUESTED** [review]
Works well and seems to fix both issues.

Blocker: the range strings are reversed. Otherwise all good.

File: packages/grafana-ui/src/components/TimePicker/TimePicker.tsx (hunk 1 above)
Comment (kaydelaney) [reply packages/grafana-ui/src/components/TimePicker/TimePicker.tsx]: oh, silly mistake, will fix

**Review by davkal — APPROVED** [review]
All good now 🎉

**Comment by davkal** [comment]
Fixes #18178 as well.

**Pull request merged**